javascript-practice/src/js/views/createNewElement.js

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
The file name should be `dom.js`. You'll join all common functions to interact with the DOM in this file